02-molecules/cards/cards.stories.js

```diff
@@ -43,3 +44,17 @@ export const cardWithBackground = () => (
     nibh pulvinar a.
   </Card>
 );
+export const newsCard = () => (
+  <NewsCard
+    image={{
+      alt: 'Random Image from Picsum',
+      src: 'https://picsum.photos/400/300.jpg',
+    }}
+    heading="This is the card heading"
+    subheading="Curabitur non nulla sit amet nisl tempus convallis quis ac lectus"
+    body="Ellentesque in ipsum id orci porta dapibus. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a. Mauris blandit aliquet elit, eget tincidunt nibh pulvinar a."
+    link_text="Read more about this card"
+    date="2119-12-09T12:00:00"
+    tag="Fun Stuff"
+  />
```

Comment:
Please add a button to the story.

Should this use the same test copy from the Twig Site? https://dev-westernuni.pantheonsite.io/storybook/?path=/story/molecules-cards--news-card

Reply:
Sure